drivers/pwm/pwm_nrf5_sw.c

```diff
@@ -114,13 +123,13 @@ static int pwm_nrf5_sw_pin_set(struct device *dev, u32_t pwm,
 	/* configure GPIO pin as output */
 	NRF_GPIO->DIRSET = BIT(pwm);
 	if (pulse_cycles == 0) {
-		/* 0% duty cycle, keep pin high (for active low LED) */
-		NRF_GPIO->OUTSET = BIT(pwm);
+		/* 0% duty cycle, keep pin low */
```

Comment:
@jhedberg does the change in polarity effect micro:bit samples?

Reply:
At least the sound sample for the micro:bit keeps working fine with this patch, so no objections from me

Reply:
ok, I would prefer this to be Kconfig option (phase). But accept the change here as-is for now.